modules/biscuit/biscuitblaster/meta.yml

```
@@ -0,0 +1,78 @@
+name: biscuit_blaster
+
+description: A fast, compact one-liner to produce duplicate-marked, sorted, and indexed BAM files using Biscuit
+keywords:
+  - biscuit
+  - DNA methylation
+  - WGBS
+  - scWGBS
+  - bisulfite sequencing
+  - aligner
+  - bam
+
+tools:
+  - biscuit:
+      description: A utility for analyzing sodium bisulfite conversion-based DNA methylation/modification data
+      homepage: https://huishenlab.github.io/biscuit/
+      documentation: https://huishenlab.github.io/biscuit/#getting-started
+      tool_dev_url: https://github.com/huishenlab/biscuit
+      doi: ""
+      licence: ["MIT"]
+  - samblaster:
+      description: |
+        samblaster is a fast and flexible program for marking duplicates in read-id grouped paired-end SAM files.
+        It can also optionally output discordant read pairs and/or split read mappings to separate SAM files,
+        and/or unmapped/clipped reads to a separate FASTQ file.
+        By default, samblaster reads SAM input from stdin and writes SAM to stdout.
+      homepage: None
+      documentation: https://github.com/GregoryFaust/samblaster
+      tool_dev_url: https://github.com/GregoryFaust/samblaster
+      doi: "10.1093/bioinformatics/btu314"
+      licence: ["MIT"]
+  - samtools:
+      description: |
+        SAMtools is a set of utilities for interacting with and post-processing
+        short DNA sequence read alignments in the SAM, BAM and CRAM formats, written by Heng Li.
+        These files are generated as output by short read aligners like BWA.
+      homepage: http://www.htslib.org/
+      documentation: hhttp://www.htslib.org/doc/samtools.html
+      doi: 10.1093/bioinformatics/btp352
+      licence: ["MIT"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
+      type: file
+      description: |
+        List of input fastq files of size 1 and 2 for single-end and paired-end data,
+        respectively.
+  - index:
+      type: dir
+      description: Biscuit genome index directory
```

```suggestion
      description: Biscuit genome index directory
```

output from ...? I say this as you specify a very highly specific pattern for the input here

---

Updated...